charts/stable/reg/values.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  repository: ghcr.io/k8s-at-home/reg
+  pullPolicy: IfNotPresent
+  tag: v0.16.1
+
+strategy:
+  type: Recreate
+
+# See more environment variables in the reg documentation
+# https://github.com/genuinetools/reg
+env:
+  EXTRA_ARGS: "server --registry r.j3ss.co"
```

Comment:
I'm not familiar with the app, but is this a good value to have as default?

Reply:
The ci tests fail if we don't specify a server to run.

Reply:
Ah, in that case you can create a `ci/ct-values.yaml` file containing this. That way the CI will use that and we can remove the default here.

Reply:
@bjw-s I'm trying to determine the best route to implement `ci/ct-values.yaml`. I'm looking at `charts-lint-test.yaml` and trying to see if I should add on to the `changes-install` and `install` jobs in `charts-lint-test.yaml`, or if I should just create a whole new job. Which would you recommend?

Reply:
You just need to place values in `ci/ct-values.yaml` that you want to run only on CI